elements/lisk-p2p/src/p2p.ts

```diff
@@ -722,8 +721,20 @@ export class P2P extends EventEmitter {
 	}
 
 	private _handleGetPeersRequest(request: P2PRequest): void {
-		// TODO later: Remove fields that are specific to the current Lisk protocol.
-		const peers = this._pickRandomPeers(MAX_PEER_LIST_BATCH_SIZE).map(
+		// TODO: Get this from peerbook
+		const knownPeers = [
+			...this._newPeers.values(),
+			...this._triedPeers.values(),
+		];
+		/* tslint:disable no-magic-numbers*/
+		const min = Math.ceil(Math.min(1000, knownPeers.length * 0.25));
```

Comment:
do you think we can have a const defined for 1000 and 100 if it makes sense? otherwise maybe its unnecessary

Reply:
I thought about it but there are 2 other numbers which would necessitate a total 4 new const. I am not sure if there would be much value added.